components/match2/wikis/sideswipe/match_group_input_custom.lua

```diff
@@ -0,0 +1,532 @@
+	local score3 = tonumber(opponent.score3 or '')
+	if score2 then
+		opponent.extradata = {
+			score2 = score2,
+			score3 = score3,
+			set1win = Logic.readBool(opponent.set1win),
+			set2win = Logic.readBool(opponent.set2win),
+			set3win = Logic.readBool(opponent.set3win),
+			additionalScores = true
+		}
+	end
+
+	return opponent
+end
+
+-- called from Module:Match/Subobjects
+function p.processPlayer(frame, player)
+	_frame = frame
+	return player
+end
+
+--
+--
+-- function to sort out winner/placements
+function p._placementSortFunction(table, key1, key2)
+	local op1 = table[key1]
+	local op2 = table[key2]
+	local op1norm = op1.status == _STATUS_HAS_SCORE
+	local op2norm = op2.status == _STATUS_HAS_SCORE
+	if op1norm then
+		if op2norm then
+			local op1setwins = p._getSetWins(op1)
+			local op2setwins = p._getSetWins(op2)
+			if op1setwins + op2setwins > 0 then
+				return op1setwins > op2setwins
+			else
+				return tonumber(op1.score) > tonumber(op2.score)
+			end
+		else return true end
+	else
+		if op2norm then return false
+		elseif op1.status == _STATUS_DEFAULT_WIN then return true
+		elseif Table.includes(_ALLOWED_STATUSES, op1.status) then return false
+		elseif op2.status == _STATUS_DEFAULT_WIN then return false
+		elseif Table.includes(_ALLOWED_STATUSES, op2.status) then return true
+		else return true end
+	end
+end
+
+function p._getSetWins(opp)
+	local extradata = opp.extradata or {}
+	local set1win = extradata.set1win and 1 or 0
+	local set2win = extradata.set2win and 1 or 0
+	local set3win = extradata.set3win and 1 or 0
+	local sum = set1win + set2win + set3win
+	return sum
+end
+
+--
+-- match related functions
+--
+function matchFunctions.readDate(matchArgs)
+	return matchArgs.date
+		and MatchGroupInput.readDate(matchArgs.date)
+		or {
+			date = MatchGroupInput.getInexactDate(globalVars:get('tournament_enddate')),
+			dateexact = false,
+		}
+end
+
+function matchFunctions.getTournamentVars(match)
+	match.mode = Logic.emptyOr(match.mode, Variables.varDefault('tournament_mode', '3v3'))
+	match.type = Logic.emptyOr(match.type, Variables.varDefault('tournament_type'))
+	match.tournament = Logic.emptyOr(match.tournament, Variables.varDefault('tournament_name'))
+	match.tickername = Logic.emptyOr(match.tickername, Variables.varDefault('tournament_ticker_name'))
+	match.shortname = Logic.emptyOr(match.shortname, Variables.varDefault('tournament_shortname'))
+	match.series = Logic.emptyOr(match.series, Variables.varDefault('tournament_series'))
+	match.icon = Logic.emptyOr(match.icon, Variables.varDefault('tournament_icon'))
+	match.icondark = Logic.emptyOr(match.iconDark, Variables.varDefault('tournament_icon_dark'))
+	match.liquipediatier = Logic.emptyOr(
+		match.liquipediatier,
+		Variables.varDefault('tournament_lptier'),
+		Variables.varDefault('tournament_tier')
+	)
+	match.liquipediatiertype = Logic.emptyOr(
+		match.liquipediatiertype,
+		Variables.varDefault('tournament_tiertype')
+	)
```

Comment:
Update these to the new standards set by the new infobox on commons.

```	Variables.varDefine('tournament_name', args.name)
	Variables.varDefine('tournament_shortname', args.shortname or args.abbreviation)
	Variables.varDefine('tournament_tickername', args.tickername)
	Variables.varDefine('tournament_icon', args.icon)
	Variables.varDefine('tournament_icondark', args.icondark or args.icondarkmode)
	Variables.varDefine('tournament_series', mw.ext.TeamLiquidIntegration.resolve_redirect(args.series or ''))

	Variables.varDefine('tournament_liquipediatier', args.liquipediatier)
	Variables.varDefine('tournament_liquipediatiertype', args.liquipediatiertype)

	Variables.varDefine('tournament_type', args.type)
```

Reply:
Are you sure you want me to update these to this? Im not seeing any other wiki that uses anything remotely close to these (specifically the args)

Reply:
I copied this from Infobox League. Use the variables that are defined by it